sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 ## 5.3.0b1 (Unreleased)
 
 ### Features Added
+- Added automatic language detection to long-running operation APIs. Pass `auto` into the document `language` hint to use this feature.
+- Added `autodetect_default_language` to long-running operation APIs. Pass as the default/fallback language for automatic language detection.
+- Added property `detected_language` to `RecognizeEntitiesResult`, `RecognizePiiEntitiesResult`, `AnalyzeHealthcareEntitiesResult`,
+  `DetectLanguageResult`, `ExtractKeyPhrasesResult`, `RecognizeLinkedEntitiesResult`, `AnalyzeSentimentResult`,
```

Comment:
is there a reason for this line break here?

Reply:
Hm no, I'll fix that